bluesky/tests/conftest.py

```diff
@@ -28,15 +22,26 @@ def clean_event_loop():
     return RE
 
 
-RE = fresh_RE
+@pytest.fixture(scope='function')
+def hw(request):
+    from ophyd.sim import hw
+    return hw()
 
 
-@pytest.fixture(scope='function')
-def motor_det(request):
-    motor = Mover('motor', {'motor': lambda x: x}, {'x': 0})
-    det = SynGauss('det', motor, 'motor', center=0, Imax=1,
-                   sigma=1, exposure_time=0)
-    return motor, det
+# vendored from ophyd.sim
+class NumpySeqHandler:
```

Comment:
do we need to vendor this?

Reply:
It might be nice to run databroker tests without ophyd, so I vendored it. Seemed less annoying than conditionally registering it based on the availability of ophyd.

Reply:
I would advocate for polishing our handling of testing-without-ophyd in 1.x....